lib/evmone/instructions_opcodes.hpp

```diff
@@ -170,7 +170,10 @@ enum Opcode : uint8_t
     OP_DELEGATECALL = 0xf4,
     OP_CREATE2 = 0xf5,
 
+    OP_CALL2 = 0xf8,
```

Comment:
`RETURNCONTRACT` is currently `0xf8` in the spec, so either calls should be `f9`, `fb`, `fc`, or `RETURNCONTRACT` should be moved to `0xfc`